skuba-update/test/os/suse/Dockerfile.builder

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensuse.org/opensuse/tumbleweed
+FROM registry.opensuse.org/opensuse/leap:15.1
```

Comment:
Maybe we should skip to `Leap 15.2` ?

Reply:
Proposed here: https://github.com/SUSE/skuba/pull/1075